scripts/deployment/publish-ppa.sh

```diff
@@ -2,29 +2,30 @@
 # SPDX-FileCopyrightText: 2022 Demerzel Solutions Limited
 # SPDX-License-Identifier: LGPL-3.0-only
 
-#exit when any command fails
-RELEASES="bionic focal jammy kinetic"
-
 set -e
-echo $VERSION > ver
-FIXED_VERSION=$(awk -F. '{ print $1"."$2$3$4"0"}' ver)
```

Comment:
You should not remove this, it's there for a reason. PPA versioning is different.

Reply:
Restored the hack with a comment to remove it with v2.